trips/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "carto-deckgl-trips-example",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
```

Comment:
ALign with other examples: missing links and remove ocular-dev-tools

Reply:
Done in this example, I'll check the rest, it's legacy